native/fluent_native/src/lib.rs

```diff
@@ -31,15 +31,19 @@ fn on_init<'a>(env: Env<'a>, _load_info: Term<'a>) -> bool {
 }
 
 #[rustler::nif]
-fn init<'a>(env: Env<'a>, lang: String, bundle_init_keyword: Vec<(rustler::types::Atom, Term)>) -> Result<Term<'a>, Error> {
+fn init<'a>(
+    env: Env<'a>,
+    lang: String,
+    bundle_init_keyword: Vec<(rustler::types::Atom, Term)>,
+) -> Result<Term<'a>, Error> {
     // Getting language
     let lang_id = match lang.parse::<LanguageIdentifier>() {
         Ok(lang_id) => lang_id,
-        Err(_e) => return Ok((atoms::error(), (atoms::bad_locale(), lang)).encode(env))
+        Err(_e) => return Ok((atoms::error(), (atoms::bad_locale(), lang)).encode(env)),
     };
     // Initializing bundle
-    let mut bundle = FluentBundle::new(&[lang_id]);
-    
+    let mut bundle = FluentBundle::new_concurrent(vec![lang_id]);
```

Comment:
How the fact that now bundle is `concurrent` affects the code? We have 2 mutexes now - one from `RwLock` and one in `FluentBundle` itself. Mb `Container` should be rewriten then to `ResourceArc<_>` instead of `ResourceArc<RwLock<_>>`?

Reply:
Maybe it could work! I'll have a go at it as soon as I have some spare time. 

Worst-case scenario, the RwLock is held back by the Mutex. I don't think it's a big issue but I'll investigate